examples/environment-variables/pages/index.js

```diff
@@ -0,0 +1,93 @@
+import styles from '../styles.module.css'
+
+// The value here will be defined in the terminal, because of server side rendering,
+// but not in the browser console
+console.log('[Server only] ENV_VARIABLE:', process.env.ENV_VARIABLE)
```

Comment:
This is generally not how we should show this example. Instead it should rely on `getStaticProps` / `getServerSideProps`